inst/@sym/unique.m

```diff
@@ -47,12 +47,17 @@
 
 
 function r = unique(A)
+  cmd = {'(A,) = _ins'
+         'if A.is_Matrix:'
+         '    return sp.Matrix([list(uniq(*_ins))]),'
+         'else:'
+         '    return _ins'};
```

Comment:
I think we normally put this `cmd` after the `nargin` input processing (but could be wrong!)

Reply:
I've checked other files in the same directory. This is indeed the convention.